pkg/kubelet/volumemanager/reconciler/reconciler_test.go

```diff
@@ -54,7 +55,8 @@ func Test_Run_Positive_DoNothing(t *testing.T) {
 	dsw := cache.NewDesiredStateOfWorld(volumePluginMgr)
 	asw := cache.NewActualStateOfWorld(nodeName, volumePluginMgr)
 	kubeClient := createTestClient()
-	oex := operationexecutor.NewOperationExecutor(kubeClient, volumePluginMgr)
+	fakeRecorder := &record.FakeRecorder{}
```

Comment:
nit: you can save a couple lines total with an object mother method for this test that just adds in the fake recorder and uses a supplied client and volume plugin manager